src/meson.build

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: Apache-2.0
+
+phosphor_dbus_interfaces = dependency('phosphor-dbus-interfaces')
+phosphor_logging = dependency('phosphor-logging')
+sdbusplus = dependency('sdbusplus')
+
+rbmc_data_sync_sources = [
+    'phosphor-rbmc-data-sync-mgr_main.cpp'
```

Comment:
That's a pretty long filename.  You prefer that over something like sync_mgr_main.cpp?

Reply:
added to PR 1